third_party/libpg_query/src_backend_parser_scan.cpp

```diff
@@ -1,5 +1,4 @@
-#line 2 "third_party/libpg_query/src_backend_parser_scan.cpp"
-#line 2 "third_party/libpg_query/scan.l"
+#line 1 "third_party/libpg_query/src_backend_parser_scan.cpp"
```

Comment:
It should not be necessary to re-generate the lexer unless you edit `scan.l` (which is rarely required). Could you revert the changes to this file?

Reply:
oops, reverted!